swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0, age=0):
+        super().__init__(category, condition, age)
```

Comment:
one thing we could do is hardcode "clothing" because as a Clothing child class, it's always supposed to be "clothing", right? So, let's not even give the user tha chance to mess that up!

```suggestion
    def __init__(self, condition=0, age=0):
        super().__init__("Clothing", condition, age)
```